distribution/src/config/log4j2.properties

```diff
@@ -63,7 +63,7 @@ appender.deprecation_rolling.name = deprecation_rolling
 appender.deprecation_rolling.fileName = ${sys:es.logs.base_path}${sys:file.separator}${sys:es.logs.cluster_name}_deprecation.json
 appender.deprecation_rolling.layout.type = ESJsonLayout
 appender.deprecation_rolling.layout.type_name = deprecation.elasticsearch
-appender.deprecation_rolling.layout.esmessagefields=x-opaque-id
+appender.deprecation_rolling.layout.esmessagefields=x-opaque-id,key
```

Comment:
it was already being set in https://github.com/elastic/elasticsearch/pull/75331/files#diff-585bbd41ac3ed6c6aa4811cecbd0d3e776af7c594e3ae8558fa5434fbfa3bcd6R38
so I thought why not have it in logs too